lib/snapshy.ex

```diff
@@ -195,6 +191,21 @@ defmodule Snapshy do
     term
   end
 
+  defp inspect_options do
+    opts = default_inspect_options()
+      |> Keyword.merge(Application.get_env(:snapshy, :serialize_inspect_options, []))
+
+    struct(Inspect.Opts, opts)
+  end
+
+  defp default_inspect_options do
+    [
+      limit: :infinity,
+      printable_limit: :infinity,
+      pretty: true
+    ]
```

Comment:
Shouldn't we make `structs: false` the default?

I think it will not introduce a regression, because:
1. All existing snapshots will pass,
2. All new snapshots will be generated with the new, more robust method,
3. When using `SNAPSHY_OVERRIDE=true` only failing snapshots will be overridden, so we won't have any unintentional snapshot updates.

Let me know what you think.

Reply:
Totally agree! :)

I haven't used this at first because I didn't want to change the original configuration the library was using (`structs: true`). 

`1:` Right.
`2:` Right.
`3:` Right.